app/services/get-breadcrumbs.js

```diff
@@ -20,6 +20,11 @@ module.exports = function (id, organisationLevel) {
     breadcrumbs.push(new Link(tree[reference].name, linkGenerator.fromReference(reference)))
     reference = tree[reference].parent
   } while (reference !== undefined)
+
+  if (breadcrumbs.length < 2) {
+    breadcrumbs = []
+  }
```

Comment:
Whats the reasoning behind this? Is this due to National level being the only breadcrumb?